src/pages/markets/components/markets/_forex.js

```diff
@@ -52,7 +52,9 @@ const Forex = ({ simple_step_content }) => {
                     )
                 }
                 Multipliers={
-                    <Multipliers market_content={is_eu ? forex_multiplier_eu : forex_multiplier} />
+                    <Multipliers
+                        market_content={is_eu_uk ? forex_multiplier_eu : forex_multiplier}
```

Comment:
Could you also change the `is_eu` to `is_eu_uk` on line 31? The requirements should be the same. @kevinw-binary

Reply:
@sean-binary  okay done